module/VuFind/src/VuFind/Net/UserIpReader.php

```diff
@@ -119,10 +119,12 @@ public function getUserIp()
                         if (!empty($parts)) {
```

Comment:
We could check for `empty($parts)` before the switch and just have return statements for `'first'` and `'last'`.